tools/test/audio/comp_run.sh

```diff
@@ -42,10 +43,11 @@ parse_args ()
     FS_IN=48000
     FS_OUT=
     VALGRIND=true
-    DEBUG=
+    DEBUG=-q
```

Comment:
forgotten debug code?

Reply:
I really hate how little context (3 lines!) Github shows by default, this is very stupid. After expanding the context above I now see the `# Defaults` comments. Apologies for the noise.